src/pretix/control/templates/pretixcontrol/pdf/index.html

```diff
@@ -379,22 +379,34 @@ <h2>{% trans "Welcome to the PDF ticket editor!" %}</h2>
                         <div class="col-sm-6">
                             <label>&nbsp;</label><br>
                             <div class="btn-group btn-group-justified" role="group">
-                                <div class="btn-group" role="group">
+                                <div class="btn-group text" role="group">
                                     <button type="button" class="btn btn-default toggling" data-action="bold">
                                         <span class="fa fa-bold"></span>
                                     </button>
                                 </div>
-                                <div class="btn-group" role="group">
+                                <div class="btn-group text" role="group">
                                     <button type="button" class="btn btn-default toggling" data-action="italic">
                                         <span class="fa fa-italic"></span>
                                     </button>
                                 </div>
-                                <div class="btn-group" role="group">
+                                <div class="btn-group textarea" role="group">
                                     <button type="button" class="btn btn-default toggling" data-action="downward"
-                                        data-toggle="tooltip" title="{% trans "Flow multiple lines downward from specified position" %}">
+                                            data-toggle="tooltip" title="{% trans "Flow multiple lines downward from specified position" %}">
                                         <span class="fa fa-caret-square-o-down"></span>
                                     </button>
                                 </div>
+                                <div class="btn-group textcontainer" role="group">
+                                    <button type="button" class="btn btn-default toggling" data-action="autoresize"
+                                            data-toggle="tooltip" title="{% trans "Automatically reduce font size to fit content" %}">
+                                        <span class="fa fa-text-height"></span>
+                                    </button>
+                                </div>
+                                <div class="btn-group textcontainer" role="group">
+                                    <button type="button" class="btn btn-default toggling" data-action="splitlongwords"
+                                        data-toggle="tooltip" title="{% trans "Allow long words to be split (preview is not accurate)" %}">
+                                        <span class="fa fa-unlink"></span>
```

Comment:
Hm, not easy to find a good icon for this. Maybe I am to pre-conditioned, but the unlink does for me indeed look like remove a link. I tried `fa-scissors` (for cutting words) and `fa-ellipsis-h` (which I liked better, as it kind of shows a long line cut into pieces – although that is also not exactly what this button does). Not really sure though that these are any better than `unlink`.